### PR TITLE
docs: Some typos and grammar mistakes.

### PR DESCRIPTION
**Pull request by amitamrutiya2210**
## 📑 Description
This pull request addresses various typos and grammatical errors throughout the codebase to enhance readability and maintain consistency. The changes primarily focus on improving the quality of the documentation and ensuring clarity in the project's content.

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

## ℹ Additional Information
During my review, I found some spelling mistakes that I fixed in this pull request. If you notice anything else that needs to be changed or have any suggestions, please let me know. I'll make sure to fix them quickly to keep everything consistent with the project's standards. Your feedback is important for keeping the project professional and well-organized.

### CODE REVIEW

**Comment by amitamrutiya2210** [comment]
Hey @AlexsJones, if you have a moment, could you please review this pull request? Because it contains code that directly affects the main documentation of K8sGPT.

File: docs/reference/cli/filters.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ Active:
 > StatefulSet
 > PrometheusConfigReport
 > ReplicaSet
-Unused: 
+Unused:
```
Comment (AnaisUrlichs) [inline docs/reference/cli/filters.md:143]: You make this update also in another PR https://github.com/k8sgpt-ai/docs/pull/95
Let's merge this one first, then the other one. Thank you

**Review by AnaisUrlichs — APPROVED** [review]
